### ISSUE TITLE
Deprecated field of type array of referenced schema not displayed as deprecated 

### ISSUE THREAD

**Issue by AlaNDGT** [rapi-doc/RapiDoc#829]
**Version**: 9.3.3

**Description**:

Specified an array field as deprecated and the sub item is a reference to a schema but currently it's not being displayed as deprecated.

<img width="151" alt="image" src="https://user-images.githubusercontent.com/3120817/193035656-9051895d-9f56-405c-90f6-7e80e340523a.png">

**Expected**:  Field displayed as deprecated

**Example OAS**:

```yaml
openapi: 3.1.0
info:
  title: test
  version: '1.0'
servers:
  - url: 'http://localhost:3000'
paths:
  /test:
    parameters: []
    get:
      summary: test
      responses:
        '200':
          description: Test
          content:
            application/json:
              schema:
                type: object
                properties:
                  foo:
                    type: array
                    deprecated: true
                    items:
                      $ref: '#/components/schemas/Foo'
      operationId: get-test
      description: Test
components:
  schemas:
    Foo:
      title: Foo
      type: object
      properties:
        id:
          type: string
```

**Comment by mrin9** [comment]
the above pull request fixxes the issue only for the schema-style `table`, this should needs to be applied to schema-style `tree`

**Comment by mrin9** [comment]
fixed in [6b1836d](https://github.com/rapi-doc/RapiDoc/commit/6b1836d4302ddbb5dbc9c513659372948f785463)
